### DISCUSSION TITLE
Error on new project with MAUI Embedding turn on

### DISCUSSION THREAD

**Discussion by WenHong-Lau** [unoplatform/uno#15837] [General]
These are the options selected in the wizard
![image](https://github.com/unoplatform/uno/assets/88780667/a6fec59a-f93c-41fd-ba7f-d4810c1ae9c0)

These are the errors I've encountered.
![image](https://github.com/unoplatform/uno/assets/88780667/81b50c72-6605-448f-9b1e-c1539c02b188)

**Comment by WenHong-Lau** [comment]
Here is the solution
[UnoApp5.zip](https://github.com/unoplatform/uno/files/14552600/UnoApp5.zip)

**Reply by jeromelaban** [reply]
@Darsh0307 can you try and see if you can repro?

**Reply by Darsh0307** [reply]
Hello @jeromelaban  

I did created a project with same specs , It working completely fine on IOS , Mac Catalyst. 

Wheareas this is giving errors on Androdi and windows. The error message is same as the one shared by @WenHong-Lau

**Reply by Darsh0307** [reply]
IOS 

https://github.com/unoplatform/uno/assets/90713481/c8fb34ad-0eba-4bdb-855e-817890e61136

Mac


https://github.com/unoplatform/uno/assets/90713481/bad82eb7-eaf8-401b-9993-979c7fdca796

**Reply by Darsh0307** [reply]
Windows :- 

<img width="1393" alt="Screenshot 2024-03-20 at 6 44 49 AM" src="https://github.com/unoplatform/uno/assets/90713481/60871653-5266-44d4-80db-e97832716b2c">


Android :- 

<img width="1434" alt="Screenshot 2024-03-20 at 6 45 11 AM" src="https://github.com/unoplatform/uno/assets/90713481/536b9816-0bc3-4200-970f-d6658c0bb901">

**Reply by jeromelaban** [reply]
Let's continue tracking here: https://github.com/unoplatform/uno.extensions/issues/2223

**Comment by SolidRockProgrammer** [comment]
Hi, This issue is a showstopper for us, any idea when it will be fixed?

**Reply by sasakrsmanovic** [reply]
Hello. We are wrapping up our 5.2 release so we won't be able to triage it before then. Once 5.2 is out, which could be this week, we will triage it. If you'd need a faster fix please reach out to me at sasha at platform dot uno. I'd love to understand what you are building too!
